roles/prune_catalog/tasks/main.yml

```diff
@@ -49,7 +49,7 @@
 
     - name: "Download stable opm client"
       vars:
-        ocp_clients_url: "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/opm-linux.tar.gz"
+        ocp_clients_url: "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/opm-linux.tar.gzs"
```

Comment:
Beto, is this a typo? I do not see the file `opm-linux.tar.gzs` with an `s` at the end

Reply:
Sorry, that was part of my test to make it fail and have the always executed. Thanks!